src/core/qgismobileapp.cpp

```diff
@@ -667,7 +667,7 @@ void QgisMobileapp::onMapCanvasRefreshed()
   disconnect( mMapCanvas, &QgsQuickMapCanvasMap::mapCanvasRefreshed, this, &QgisMobileapp::onMapCanvasRefreshed );
   if ( !mProjectFilePath.isEmpty() )
   {
-    if ( !QFileInfo::exists( QStringLiteral( "%1.jpg" ).arg( mProjectFilePath ) ) )
+    if ( !QFileInfo::exists( QStringLiteral( "%1.png" ).arg( mProjectFilePath ) ) )
```

Comment:
@domi4484 , I think we need to keep the .jpg in there, otherwise people will start to see their old preview JPG appear which isn't nice UX.

Reply:
@domi4484 , sorry, my comment was wrong here. Merging.